go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/lyft/flyteplugins
+
+go 1.13
+
+require (
+	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.1.3
+	github.com/aws/aws-sdk-go v1.28.9
+	github.com/coocood/freecache v1.1.0
+	github.com/go-test/deep v1.0.5
+	github.com/golang/protobuf v1.3.2
+	github.com/hashicorp/golang-lru v0.5.4
+	github.com/json-iterator/go v1.1.9 // indirect
+	github.com/lyft/flyteidl v0.17.0
+	github.com/lyft/flytestdlib v0.3.0
+	github.com/magiconair/properties v1.8.1
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.3.0
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
+	google.golang.org/grpc v1.26.0
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/klog v1.0.0 // indirect
+	k8s.io/utils v0.0.0-20200122174043-1e243dd1a584 // indirect
+	sigs.k8s.io/controller-runtime v0.4.0
```

Comment:
We used to depend on a SHA, is this version inclusive?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.4.0 includes sha 40070e2a1958c3d974ba95da883a2bd088137789